static/app/components/group/releaseStats.spec.tsx

```diff
@@ -8,12 +9,12 @@ import GroupReleaseStats from 'sentry/components/group/releaseStats';
 describe('GroupReleaseStats', function () {
   const organization = Organization();
   const project = ProjectFixture();
-  const group = TestStubs.Group();
+  const group = GroupFixture();
 
   beforeEach(() => {
     MockApiClient.addMockResponse({
       url: `/organizations/${organization.slug}/issues/${group.id}/first-last-release/`,
-      body: {firstRelease: group.firstRelease, lastRelease: group.lastRelease},
+      body: {id: group.id, firstRelease: undefined, lastRelease: undefined},
```

Comment:
`firstRelease` and `lastRelease` don't exist on `Group`, they only exist in the response of this endpoint. Pulling them off `Group` resulted in `undefined` anyway, so I made it explicit